docs/examples/vector_stores/DocArrayHnswIndexDemo.ipynb

```diff
@@ -0,0 +1,278 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "daff0844-6030-4c75-9847-0c86424db9f7",
+   "metadata": {},
+   "source": [
+    "# DocArray Hnsw Vector Store\n",
+    "\n",
+    "[DocArrayHnswVectorStore](https://docs.docarray.org/user_guide/storing/index_hnswlib/) is a lightweight Document Index implementation provided by [DocArray](https://github.com/docarray/docarray) that runs fully locally and is best suited for small- to medium-sized datasets. It stores vectors on disk in hnswlib, and stores all other data in SQLite.\n",
```

Comment:
The first link doesn't seem to work btw

Reply:
the docs are down for some reason, will be fixed tomorrow